ui/pages/confirmations/components/confirm/footer/footer.tsx

```diff
@@ -18,6 +19,129 @@ import {
 } from '../../../../../store/actions';
 import { confirmSelector } from '../../../selectors';
 import { getConfirmationSender } from '../utils';
+import useAlerts from '../../../../../hooks/useAlerts';
+import { MultipleAlertModal } from '../../../../../components/app/confirmations/alerts/multiple-alert-modal';
+import { AlertModal } from '../../../../../components/app/confirmations/alerts/alert-modal';
+
+function ConfirmAlertModal({
+  alertKey,
+  ownerId,
+  handleCloseModal,
+  hasUnconfirmedAlerts,
+  alertModalVisible,
+  frictionModalVisible,
+  handleOpenModal,
+  onCancel,
+  onSubmit,
+}: {
+  alertKey: string;
+  ownerId: string;
+  handleCloseModal: () => void;
+  hasUnconfirmedAlerts: boolean;
+  alertModalVisible: boolean;
+  frictionModalVisible: boolean;
+  handleOpenModal: () => void;
+  onCancel?: () => void;
+  onSubmit?: () => void;
+}) {
+  if (alertModalVisible) {
+    return (
+      <MultipleAlertModal
```

Comment:
I assumed we'd never need a `MultipleAlertModal` here since we always want the review alerts option, even if they've already acknowledged all the alerts, as we want to provide the final checkbox.

Though that modal would have a button that would display the multiple alert modal.

Reply:
Perfect, removed the `MultipleAlertModal` from the footer component.